.github/workflows/ci.yml

```diff
@@ -1,45 +1,61 @@
 name: CI
 on:
   push:
-    branches: ["main"]
-    tags: ["v*"]
+    branches: [ "main" ]
   pull_request:
-    branches: ["*"]
+  release:
+    types: [ 'published' ]
```

Comment:
Why change this? I don't think I've ever had an issue with the `tags:` setting in the dozens of projects I released on github

Reply:
It follows the intended semantics: You want to run the action when you publish a release on GitHub.

In all likelihood, you don't want to run the action when you push a tag `various-fixes` or whatever, right?